redfish/chassis.go

```diff
@@ -187,6 +187,8 @@ type Chassis struct {
 	// Location shall contain location information of the
 	// associated chassis.
 	Location common.Location
+	// An indicator allowing an operator to physically locate this resource. (v1.14+)
```

Comment:
Could you add a note to the comment for [IndicatorLED](https://github.com/stmcginnis/gofish/blob/main/redfish/chassis.go#L185) noting that it is deprecated and this field should be checked instead?